blocks/init/src/blocks/components/hamburger/assets/index.js

```diff
@@ -0,0 +1,6 @@
+import { hamburgerInit, closeMobileMenuOnOverlayClick } from './hamburger';
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
rewrite this to use dynamic import 
https://github.com/infinum/wordpress-infinum-web/tree/staging/wp-content/themes/infinum/src/blocks/components/header/assets

also replace document.addEventListener('DOMContentLoaded',  with 
https://github.com/WordPress/gutenberg/tree/master/packages/dom-ready